### DISCUSSION TITLE
RCE Attacks

### DISCUSSION THREAD

**Discussion by frepog** [community/community#29528] [Other Feature Feedback, Questions, & Ideas]
Why every exploit almost is created with python
can RCE attacks be done with c

**Comment by airtower-luna** [comment]
Any language works as long as you can send the right signals to exploit whatever vulnerability exists. :wink: 

Python is popular for any kind of research and related work because as a powerful script language it works well for quick prototyping. No long compile/run/debug cycles, just write and run.

Also, if you care about avoiding exploitable issues in your own code… Avoid C if possible. For compiled languages Rust or Go are modern, safer alternatives.
